src/hw/actions/app.js

```diff
@@ -263,7 +264,7 @@ const reducer = (state: State, e: Event): State => {
         allowOpeningRequestedWording: null,
         allowManagerGranted: false,
         allowManagerRequestedWording: null,
-        requiresAppInstallation: { appName: e.appName },
+        requiresAppInstallation: { appNames: e.appNames },
```

Comment:
this requires a change on UI, correct? should we temporarily still have a `appName: e.appNames[lastIndex]` ?

Reply:
This would definitely need a minor adaptation of the UI, but it's aligned with the redesign that we discussed today with @cthepot-ledger, I think passing appName temporarily to not break at all can be fine if you deem it necessary